paths/product_tests.yaml

```diff
@@ -2,9 +2,19 @@ x-swagger-router-controller: mock
 get:
   description: |
     Request all available Product Tests
-  tags:
-    - ProductTest
   parameters:
+    - name: filter
+      in: query
+      description: Filter product tests; returns all by default
+      required: false
+      schema:
+        type: string
+        enum:
```

Comment:
would `all` technically be part of this enum, even when it's the default?

Reply:
Not really, because `all` is not one of the `case` options in core-logic. Do you think I should add it anyways? Wondered about this too

Reply:
it is internally still the default, so I would include it, just so the default is part of the possible values collection